schema.json

```
@@ -280,8 +280,12 @@
             "type": "string"
         },
         "date": {
+            "examples": [
```

```suggestion
            "$comment": "Note to tool implementers: it is necessary to cast YAML 'date' objects to string objects when validating against this schema.",
            "examples": [
```

Add comment as discussed.